client/components/telecom/telephony/alias/records/telecom-telephony-alias-records.html

```diff
@@ -330,7 +330,9 @@
                     <tbody data-ng-if="$ctrl.records.isLoading">
                         <tr>
                             <td colspan="7">
-                                <oui-loader data-size="m"></oui-loader>
+                                <div class="text-center">
+                                    <oui-spinner></oui-spinner>
```

Comment:
+`data-size="s"`

Reply:
Mmmmh... Indead spinner size into array should be size s. But this rule will probably be applied later with `oui-datagrid` implementation ?